config/routes.rb

```diff
@@ -49,5 +49,6 @@
   get '/assets/signs/:width-:height/*filename' => 'sign_image#show', :width => /\d+/, :height => /\d+/, :format => false
   get '/sitemap.xml' => 'sitemaps#index', defaults: { format: 'xml' }, as: :sitemap
 
+  get '/random_sign' => 'pages#random_sign'
```

Comment:
maybe we should call it 'daily_sign' because it doesn't change enough to be a method to request for a random sign

Reply:
`/random_sign` is the destination when you hit the "Show me another sign" button on the homepage so it really is a random sign. The homepage will use the cached version but `/random_sign` will give you a new sign every time. @DSRU wants to keep the "Show me another sign" functionality and we still want the homepage to be fast/cached so I implemented it this way